api/git/git_integration_test.go

```diff
@@ -25,3 +25,16 @@ func TestService_ClonePrivateRepository_GitHub(t *testing.T) {
 	assert.NoError(t, err)
 	assert.FileExists(t, filepath.Join(dst, "README.md"))
 }
+
+func TestService_LatestCommitID_GitHub(t *testing.T) {
+	ensureIntegrationTest(t)
+
+	pat := getRequiredValue(t, "GITHUB_PAT")
```

Comment:
```suggestion
	path := getRequiredValue(t, "GITHUB_PATH")
```
?

Reply:
PAT is an acronym for Personal Access Token, so no typo here.

Reply:
:+1:

maybe give it a clearer name? accessToken? personalAccessToken?